### PR TITLE
Sync changes from main repo

### PR DESCRIPTION
**Pull request by mresposito**
# New

# Testing

- [ ] Unit
- [ ] Integration
- [ ] Localhost

# Screenshots

### CODE REVIEW

File: packages/examples/basic/components/Header.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import {
   Box,
   Typography,
   Grid,
-  Button as MaterialButton
+  Button as MaterialButton,
 } from "@material-ui/core";
-import { useEditor } from "@candulabs/craft-core";
+import { useEditor } from "@craftjs/core";
```
Comment (matdru) [inline packages/examples/basic/components/Header.js]: We shouldn't go back to offical name i think?

File: packages/examples/basic/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0-beta.3",
   "private": true,
   "dependencies": {
-    "@candulabs/craft-core": "^0.1.0-beta.3.2",
+    "@candulabs/craft-core": "^0.1.0-beta.3",
```
Comment (matdru) [inline packages/examples/basic/package.json]: This needs to be ^0.1.0-beta.3.2 at the moment

File: packages/examples/landing/package.json [hunk 3]
```diff
@@ -9,8 +9,8 @@
     "clean": "rimraf lib .next out dist"
   },
   "dependencies": {
-    "@candulabs/craft-core": "^0.1.0-beta.3.2",
-    "@candulabs/craft-layers": "^0.1.0-beta.2",
+    "@candulabs/craft-core": "^0.1.0-beta.3",
+    "@candulabs/craft-layers": "^0.1.0-beta.3",
```
Comment (matdru) [inline packages/examples/landing/package.json]: "@candulabs/craft-core": "^0.1.0-beta.3.2",
"@candulabs/craft-layers": "^0.1.0-beta.2",

Comment (mresposito) [reply packages/examples/landing/package.json]: soo confusing

Comment (mresposito) [reply packages/examples/landing/package.json]: but fixed

**Pull request merged**